cmd/compose/up.go

```diff
@@ -79,6 +80,16 @@ func (opts upOptions) apply(project *types.Project, services []string) (*types.P
 	return project, nil
 }
 
+func (opts *upOptions) validateNavigationMenu(experimentals *experimental.State) {
+	if !opts.navigationMenuChanged {
+		opts.navigationMenu = SetUnchangedOption(ComposeMenu, experimentals.NavBar())
+	}
+
+	if opts.navigationMenu && !streams.NewIn(os.Stdout).IsTerminal() {
```

Comment:
you should access dockerCli.Out() directly here

Reply:
changed to `dockerCli.Out().isTerminal`